### ISSUE TITLE
RubberDucky Keyboard

### ISSUE THREAD

**Issue by lyfak** [Marten4n6/EvilOSX#86]
Is it posible to change the default keyboard layout ?

**Comment by shizonic** [comment]
Yes, use the newest git version: https://github.com/hak5darren/USB-Rubber-Ducky/blob/master/Encoder/README#L1
